Jenkinsfile

```diff
@@ -26,12 +26,11 @@ pipeline {
                 }
             }
         }
-        stage('Build ntsc-1.2, check disasm metadata') {
+        stage('Build ntsc-1.2') {
             steps {
                 sh 'ln -s /usr/local/etc/roms/oot-ntsc-1.2-us.z64 baseroms/ntsc-1.2/baserom.z64'
                 sh 'make -j$(nproc) setup VERSION=ntsc-1.2'
                 sh 'make -j$(nproc) VERSION=ntsc-1.2'
-                sh '.venv/bin/python3 tools/check_disasm_metadata_unksyms.py -v ntsc-1.2'
                 sh 'make clean assetclean VERSION=ntsc-1.2'
```

Comment:
shouldnt we wait until this matches upstream to do this?

Reply:
well I don't personally use the disassembly anymore now that there's a way to get a matching build. But you guys seem to be doing a lot of renames so I figured this doesn't pass the cost/benefit test

Reply:
I just mean the jenkins build specifically. should we comment the task out or something until its fully matching and it has a purpose to run?

Reply:
oh I see. Right now this checks that the ntsc-1.2 build isn't broken at least which is kind of nice but maybe not worth the extra build time

Reply:
thats fine to me, i thought it had 0 value rn